README.md

```diff
@@ -17,7 +17,7 @@ $ p2r https://hbr.org/2019/11/getting-your-team-to-do-more-than-meet-deadlines
 ```
 
 The script can be run through the ``p2r`` command line program or via Docker 
-(see below).
+(see below). If you're using MacOS, we recommend using the [Alfred workflow](#alfred).
```

Comment:
I'd prefer to write "you might also like" instead of "recommend", I'm not a mac user and can't verify (although it does look handy!)